src/syscalls/platform_mock.rs

```diff
@@ -9,18 +16,27 @@ pub unsafe fn subscribe(
     unimplemented()
 }
 
-pub unsafe fn command(_: usize, _: usize, _: usize, _: usize) -> isize {
+pub fn command(_: usize, _: usize, _: usize, _: usize) -> isize {
```

Comment:
*Maybe* `command`s in the `raw` module should stay `unsafe`, s.t. the only place where `unsafe` is converted to *safe* is the `syscalls` module. What do you think?

Reply:
Mh... I think the `raw` command is exactly as safe as the `syscall` command and so I would skip the `unsafe`.

Reply:
I considered the `platform_x` modules to be a thin platform-specific abstraction over the asm calls. The knowledge that commands are actually safe is better centralized in the higher-level non-raw module. Besides that, making `raw::command` safe introduces more `unsafe` which we should avoid.

Reply:
I agree that the fact that commands are safe is more connected to the fact that it is a tock syscall than to the specfic architecture-specific asm-code. So I fixed this

bors try